hooks/shellcheck.sh

```diff
@@ -8,17 +8,47 @@ set -e
 export PATH=$PATH:/usr/local/bin
 
 exit_status=0
+ENABLE_LIST=""
 
-for file in "$@"; do
- if (head -1 "$file" |grep '^#!.*sh'>/dev/null); then
+# Arguments
+parse_arguments() {
+  while [ $# -gt 0 ]; do
+    # Get param and value using parameter expansion, splitting on = or " "
+    param="${1%[ =]*}"
+    value="${1#*[ =]}"
+    if [ "$param" = "$value" ]; then value="$2"; fi
```

Comment:
We follow [google's style guide for bash](https://google.github.io/styleguide/shellguide.html#s6.3-tests). In practice, this means using `[[]]` and `==`:

```suggestion
    if [[ "$param" == "$value" ]]; then value="$2"; fi
```